htmresearch/frameworks/classification/utils/sensor_data.py

```diff
@@ -33,24 +33,23 @@
 
 
 
-def plotSensorData(expResults):
+def plotSensorData(inpuFilePaths, markers=True, categoryLabels=None):
   """
   Plot several sensor data CSV recordings and highlights the sequences.
-  @param expResults: (list of dict) list of results for each experiment
   """
 
-  categoryColors = ['grey', 'b', 'r', 'y']
+  categoryColors = ['grey', 'blue', 'yellow', 'red', 'green', 'orange',
```

Comment:
I was using that previously, but it was a bit hard to distinguish adjacent colors, so I ended up using a fixed set.